code/LeftAndMain.php

```diff
@@ -1099,11 +1110,11 @@ protected function getSearchFilter()
         // Validate classname
         $filterClass = $params['FilterClass'];
         $filterInfo = new ReflectionClass($filterClass);
-        if (!$filterInfo->implementsInterface('SilverStripe\\Admin\\LeftAndMain_SearchFilter')) {
+        if (!$filterInfo->implementsInterface(LeftAndMain_SearchFilter::class)) {
```

Comment:
I optimised the code to be more performant when getSearchFilter() is requested more often, as the new rewrite does. At the same time I cleaned up code quality.